### PR TITLE
Ti 3197 drag&drop of user's booking, option of visible timeline

### PR DESCRIPTION
**Pull request by eDombr**
https://thing-it.atlassian.net/browse/TI-3197

### CODE REVIEW

File: projects/thing-it-common-lib/src/lib/booking-timeline/booking-timeline.component.html [hunk 1]
```diff
@@ -15,11 +15,11 @@
...
         'left.em': findPositionFromHours(bookingTemp.start),
         'width.em': bookingWidth
-      }">
+      }"
+      #userBooking>
       <div class="timelineButtonOverlay"
-        (pan)="onPan($event)"
-        (panend)="onPanEnd()">
-        <div class="timelineButton"></div>
+        #handle>
```
Comment (YurySakovich) [inline projects/thing-it-common-lib/src/lib/booking-timeline/booking-timeline.component.html]: I think not clear naming

File: projects/thing-it-common-lib/src/lib/booking-timeline/booking-timeline.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="timelineWrapper" #timeline>
-  <div class="timeline">
+  <div class="timeline" [style.width.em]="timelineScale.length*4">
```
Comment (YurySakovich) [inline projects/thing-it-common-lib/src/lib/booking-timeline/booking-timeline.component.html]: magic number

File: projects/thing-it-common-lib/src/lib/thing-it-common-lib.component.ts [hunk 3]
```diff
@@ -42,6 +40,7 @@ export class ThingItCommonLibComponent implements OnInit {
 
   public changeBooking(booking: BookingTimeline): void {
     this.booking = booking;
+    console.log(booking);
```
Comment (YurySakovich) [inline projects/thing-it-common-lib/src/lib/thing-it-common-lib.component.ts]: remove

File: projects/thing-it-common-lib/src/lib/booking-timeline/booking-timeline.component.html (hunk 2 above)
Comment (eDombr) [reply projects/thing-it-common-lib/src/lib/booking-timeline/booking-timeline.component.html]: fixed

File: projects/thing-it-common-lib/src/lib/thing-it-common-lib.component.ts (hunk 3 above)
Comment (eDombr) [reply projects/thing-it-common-lib/src/lib/thing-it-common-lib.component.ts]: removed

**Pull request merged**